src/controller/postRouter.js

```diff
@@ -86,6 +94,28 @@ postRouter.patch("/:id/likes", authenticate, postMiddleware.validateLike, async
     }
 });
 
+postRouter.delete("/:postId", postOwnerOrAdminAuthenticate, async (req, res) => {
```

Comment:
Since we have descriptions for endpoints now, it'd be good to add descriptions to these endpoints with required request properties and potential responses.